package.json

```diff
@@ -4,12 +4,14 @@
   "description": "Launch rule engine which processes rules on client websites and delegates logic to extensions.",
   "license": "Apache-2.0",
   "scripts": {
-    "build": "rollup -c --intro \"if (!window.atob) { console.warn('Adobe Launch is unsupported in IE 9 and below.'); return; }\" && uglifyjs dist/engine.js -c -m -o dist/engine.min.js",
-    "build:watch": "rollup --watch -c --intro \"if (!window.atob) { console.warn('Adobe Launch is unsupported in IE 9 and below.'); return; }\" && uglifyjs dist/engine.js -c -m -o dist/engine.min.js",
+    "build": "npm run clean && rollup -c && npm run minify",
+    "build:watch": "npm run clean && rollup --watch -c",
```

Comment:
removed the `minify` step here because in `watch` mode it wasn't working as expected anyways.